server/src/sass/components/_flood-status.scss

```diff
@@ -39,61 +64,89 @@
     .defra-flood-status-item__icon {
         display: table-cell;
         vertical-align: middle;
-        padding:7.5px;
+        padding:7.5px 10px;
         @include mq ($from: tablet) {
             padding:10px 12.5px;
         }
     }
-    .defra-flood-status-item__icon:after {
-        display: block;
-        content:'';
-        background: transparent url('#{$govuk-images-path}icon-flood-symbols.svg') no-repeat;
-        background-size: 80px 200px;
-        width: 40px;
-        height: 40px;
-        @include mq ($from: tablet) {
-            background-size: 100px 250px;
-            width: 50px;
-            height: 50px;
-        }
+    &:after {
+        position: absolute;
+        box-sizing: border-box;
+        content: '';
+        left: 0px;
+        top: 0px;
+        bottom: 0px;
+        right: 0px;
+        border: 1px solid transparent;
+        pointer-events: none;
     }
 }
 .defra-flood-status-item--severe {
-    background-color: mix(white, #e3000f, 93%);
+    background-color: govuk-tint(#e3000f, 93);
```

Comment:
hex colours in here should probably be extracted to sass variables to be more DRY